docs/accuracy_aware_model_training/AdaptiveCompressionLevelTraining.md

```diff
@@ -28,6 +39,19 @@ To launch the adaptive compression training loop, the user is expected to define
-The default behavior for the compression rate search algorithm implies changes in the compression rate level value by a step value that is decreasing throughout training. The training is terminated once the compression rate step value reaches the minimal value determined by the `"minimal_compression_rate_step"` parameter that can be specified in the `"accuracy_aware_training"` section of the config (default value is 0.025). The initial value for the compression rate step is given be the `"compression_rate_step"` parameter and is equal to 0.1 by default. The step value is decreased by the `"step_reduction_factor"` value (0.5 by default) at points throughout training whenever the direction of change in compression rate changes at a point where the new compression rate is selected. That is, if a too big of an increase in compression rate resulted in the accuracy metrics below the user-defined criterion, the compression rate is reduced by a lower step in an attempt to restore the accuracy and vice versa, if the decrease was sufficient to satisfy the accuracy criterion, the compression rate is increased by a lower step to check if this higher compression rate could also result in tolerable accuracy values. This sequential search is limited by the minimal granularity of the steps given by `"minimal_compression_rate_step"`.
+
+## Description of the work of Adaptive Compression Level training loop
+
+The first step is **Initial Training Phase** - It corresponds to the amount of epochs that the model is going to be trained for with the initial compression rate level/schedule set by the user in the standard NNCF manner (the initial pruning rate schedule above is an exponential schedule with the target pruning rate of 0.1).
+
+The second one is **Finding the optimal compression rate**, where the next compression rate value is determined by the search algorithm and the model is fine-tuned with that selected compression rate value for `"patience_epochs"` number of epochs. The process is continued until after the search algorithm terminates. The returned model is the model with the highest compression rate encountered during training given that is satisfies the accuracy drop criterion -- the accuracy value of the compressed model should not be more that `"maximal_relative_accuracy_degradation`" percent less that the original uncompressed model's accuracy value.
```

Comment:
The process is continued until after the search algorithm terminates =>
The process is continued until the search algorithm terminates

Reply:
The following is really hard to understand. Please rephrase it.
"the accuracy value of the compressed model should not be more that "maximal_relative_accuracy_degradation" percent less that the original uncompressed model's accuracy value"

Reply:
Rephrased, please, take a look